### PR TITLE
Add locales for en-US and other.

### PR DESCRIPTION
**Pull request by shenst1**
@joshreeves

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -5,4 +5,8 @@ module.exports = {
     NEXT_EXAMPLE_CMS_DATOCMS_API_TOKEN:
       process.env.NEXT_EXAMPLE_CMS_DATOCMS_API_TOKEN,
   },
+  i18n: {
+    locales: ['en-US', 'en-UK'],
```
Comment (shenst1) [inline next.config.js:9]: `en-UK` could be anythikng. It is a placeholder for all others besides `en-US`

File: pages/[slug].js [hunk 2]
```diff
@@ -7,21 +7,29 @@ import { pageFragment, layoutFragment} from "../lib/fragments";
 
 export async function getStaticPaths() {
   const data = await request({ query: `{ allPages(first: 100) { slug } }` });
-
+  const paths = data.allPages.flatMap((p) => ([
+    {
```
Comment (shenst1) [inline pages/[slug].js:11]: this generates the pages needed for each locale

File: pages/[slug].js [hunk 3]
```diff
@@ -7,21 +7,29 @@ import { pageFragment, layoutFragment} from "../lib/fragments";
...
-
+  const paths = data.allPages.flatMap((p) => ([
+    { 
+      params: { slug: p.slug }, 
+      locale: 'en-US' 
+    },
+    {
+      params: { slug: p.slug }
+    }
+  ]))
```
Comment (joshreeves) [inline pages/[slug].js:18]: Could trim these down with shorthand objects.
```
const paths = data.allPages.flatMap({ slug } => ([
  {
    params: { slug },
    locale: 'en-US'
  },
  {
    params: { slug }
  }
])
```

**Review by joshreeves — COMMENTED** [review]
Looks good. One minor stylistic comment. Otherwise 🚢

**Pull request merged**